README.md

```diff
@@ -122,8 +121,9 @@ await ln.boost(boost);
 
 Nostr is a simple, open protocol that enables truly censorship-resistant and global value-for-value publishing on the web. Nostr integrates deeply with Lightning. [more info](https://nostr.how/)
 
-alby-tools provides helpers to create [zaps](https://github.com/nostr-protocol/nips/blob/master/57.md)
+alby-tools provides helpers to create [zaps](https://github.com/nostr-protocol/nips/blob/master/57.md).
 
+Please note users need an extension that implements [NIP-07](https://github.com/nostr-protocol/nips/blob/master/07.md) (to sign the zap request) and [WebLN](webln.guide) (to pay the invoice) for seamless zapping.
```

Comment:
I think it is good to have an example in the README because zapping is one of the great features of alby-tools.

The 1st example (the one in the README) relies on a global webln object existing, right?

The 2nd, larger example shows how to provide a custom webln object.

Reply:
@bumi I simplified the example in the README and added a new zaps example js file